app/views/course/assessment/assessments/authenticate.html.slim

```diff
@@ -1,10 +1,23 @@
+- locked = assessment_not_unlocked(@assessment.time_for(current_course_user))
```

Comment:
This is not so much "locked" than it is "not_started"

Reply:
Done!